css/schedule.scss

```diff
@@ -4,12 +4,18 @@
   text-align: center;
 }
 
-.section__header {
+.schedule__header {
   font-size: 30px;
   text-align: center;
   font-weight: 400;
   margin-bottom: 20px;
   margin-top: 20px;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
 
   &:hover {
```

Comment:
can you make this cursor: pointer on hover? and have some transition (maybe 250ms)